randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -930,8 +930,14 @@ Extra - room_id: [27]
   * L0 Hatch to Big Shell 2/Door to Big Shell 1
   * Extra - door_idx: (56,)
   > Door to Shell Access
-      # TODO: Add jump extend and single wall jump template options after confirmation. 98% sure this is doable. -Herc.
-      Space Jump and Can Use Power Bombs
+      All of the following:
+          Can Use Power Bombs
+          Any of the following:
+              Space Jump
+              # Possible with Single Wall Jumping Physics going up the left side.
+              Wall Jump (Intermediate) and Can Single Walljump
```

Comment:
With Hi-Jump this could probably be bumped down to beginner.
NHJ can stay intermediate though.